src/NuGet.Core/NuGet.Commands/RestoreCommand/LockFileUtils.cs

```diff
@@ -295,5 +344,129 @@ private static bool GroupHasNonEmptyItems(IList<LockFileItem> group)
         {
             return group?.Any(item => !item.Path.EndsWith(PackagingCoreConstants.ForwardSlashEmptyFolder)) == true;
         }
+
+        /// <summary>
+        /// Group all items by the primary key, then select the nearest TxM 
+        /// within each group.
+        /// Items that do not contain the primaryKey will be filtered out.
+        /// </summary>
+        private static List<ContentItemGroup> GetContentGroupsForFramework(
+            LockFileTargetLibrary lockFileLib,
```

Comment:
I meant to ask where `LockFileTargetLibrary lockFileLib` is used (and if this is removed the caller method can have this removed as well I think).